src/sentry/static/sentry/app/views/releaseArtifacts.jsx

```diff
@@ -125,23 +125,27 @@ const ReleaseArtifacts = React.createClass({
         <div className="panel panel-default">
           <div className="panel-heading panel-heading-bold">
             <div className="row">
-              <div className="col-sm-8 col-xs-7">{'Name'}</div>
-              <div className="col-sm-2 col-xs-2">{'Size'}</div>
-              <div className="col-sm-2 col-xs-3 align-right" />
+              <div className="col-lg-7 col-sm-6">{'Name'}</div>
+              <div className="col-lg-2 col-sm-2">{'Distribution'}</div>
```

Comment:
We could change this (hide it) later. But if we keep it we need to explain what this means and how to use it.